recipe/meta.yaml

```
@@ -331,7 +375,8 @@ outputs:
         # the whole test suite takes forever, but we should get a good enough coverage
         # for potential packaging problems by running a fixed subset
         - export OMP_NUM_THREADS=4  # [unix]
-        - python -m pytest -n auto {{ tests }} -k "not ({{ skips }})" --durations=50    # [not (aarch64 and cuda_compiler_version != "None")]
+        - python -m pytest -n auto {{ tests }} -k "not ({{ skips }})" --durations=50    # [not win and not (aarch64 and cuda_compiler_version != "None")]
+        - python -m pytest {{ tests }} -k "not ({{ skips }})" --durations=50            # [win and not (aarch64 and cuda_compiler_version != "None")]
```

```suggestion
        - python -m pytest -s {{ tests }} -k "not ({{ skips }})" --durations=50            # [win and not (aarch64 and cuda_compiler_version != "None")]
```

`-s` should help getting the actual error message from Python.